### PR TITLE
5KU-ENHANCEMENT - Customized scrollbars added

### PR DESCRIPTION
**Pull request by krazziekay**
**What's in this PR?**
- Added component for showing custom scrollbars.
- Removed the gradient logic

**Why**
- Better UI for users with long list of orgs

**Test**
- Added test case for error states
- Tested in staging

**Video**

https://github.com/atlassian/github-for-jira/assets/13076255/0d06238b-8c01-402d-a27f-edebb17f935a

### CODE REVIEW

File: spa/src/pages/ConfigSteps/OrgsContainer/index.tsx [hunk 1]
```diff
@@ -103,66 +81,68 @@ const OrganizationsList = ({
 		}
 	};
 	return (
-		<>
-			<div css={orgsWrapperStyle} ref={containerRef}>
+		<Scrollbars
+			style={{
+			maxHeight: 250,
```
Comment (rachellerathbone) [inline spa/src/pages/ConfigSteps/OrgsContainer/index.tsx:86]: Nit: can we avoid using magic numbers here and opt for constants? Will just make the code a little more readable.

File: spa/src/pages/ConfigSteps/OrgsContainer/index.tsx [hunk 2]
```diff
@@ -103,66 +81,68 @@ const OrganizationsList = ({
...
-											}}
-										>
-											Connect
-										</Button>
-									)}
-								</>
-							) : (
-								<>
-									<div>
+						const hasError = !canConnect(org);
```
Comment (rachellerathbone) [inline spa/src/pages/ConfigSteps/OrgsContainer/index.tsx:93]: Can we add some tests for this component? Don't think we need anything too crazy but we should at least have some for canConnect with different inputs to ensure it returns the correct result based on the organization's properties (requiresSsoLogin, isIPBlocked, isAdmin).

Comment (krazziekay) [reply spa/src/pages/ConfigSteps/OrgsContainer/index.tsx:97]: Added test cases for the error conditions.

File: spa/src/pages/ConfigSteps/OrgsContainer/index.tsx (hunk 1 above)
Comment (krazziekay) [reply spa/src/pages/ConfigSteps/OrgsContainer/index.tsx]: Done

**Review by rachellerathbone — APPROVED** [review]

**Pull request merged**